StripeFinancialConnections/StripeFinancialConnections/Source/Native/PartnerAuth/PartnerAuthViewController.swift

```diff
@@ -637,6 +634,28 @@ final class PartnerAuthViewController: SheetViewController {
                 pane: .partnerAuth
             )
     }
+
+    // Defensive programming to avoid completing the same auth session twice.
+    //
+    // There's been a series of odd edge-cases where the same auth session
+    // could complete twice, so this acts as future-proofing that this
+    // never happens again.
+    private var lastCompletedAuthSessionId: String?
```

Comment:
not proud of "patches" like this but this pane has had a history of mysterious edge-cases so this safe guard is a good idea